.travis.yml

```diff
@@ -0,0 +1,9 @@
+matrix:
+  include:
+  - language: python
+    python:
+      "3.8.2"
+    install:
+      - "pip install -r requirements.txt"
+    script:
+      - "python -m unittest discover tests"
```

Comment:
@ethanschreur please add a before script that configures the test database. This database will be used to run all tests.